_includes/email-subscribe-form.html

```diff
@@ -8,15 +8,32 @@
       enctype="application/x-www-form-urlencoded">
 
     <div class="form-group">
-        <label class="u-visually-hidden" for="email-subscribe-form_email">
+        <label for="email-subscribe-form_email">
             Email address
         </label>
         <input class="reveal-on-focus_target"
                id="email-subscribe-form_email"
                type="email"
                name="email"
-               placeholder="Your email">
+               placeholder="example@mail.com">
     </div>
+    {# Uncomment this and remove condition to test
```

Comment:
Yes, both are still needed. The idea is to use one subscribe form instead of duplicating it just for events. The condition will eventually need to be wired to testing whether the page is an event page, but that wasn't really possible until we have some real event pages.